style.css

```
@@ -0,0 +1,74 @@
+.body{
+    font:
+}
+.header{
+    background-color: #374D26;
+    color: white;
+    height:70px;
+}
+
+
+.header1 {
+    font-size:30px;
+    padding:30px 20px;
```

下記のようにしてあげると良いです！あるいはflexboxを使うのもありです！
```suggestion
    line-height: 70px;
    max-width: 1440px;
    margin: auto;
```